src/persist/zcl_abapgit_persistence_db.clas.abap

```diff
@@ -160,6 +163,14 @@ CLASS zcl_abapgit_persistence_db IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD list_by_keys.
+    FIELD-SYMBOLS: <ls_key> LIKE LINE OF it_keys.
+    LOOP AT it_keys ASSIGNING <ls_key>.
+      SELECT * FROM (c_tabname) WHERE value = @<ls_key> APPENDING TABLE @rt_contents.
```

Comment:
downport,
```suggestion
      SELECT * FROM (c_tabname) WHERE value = <ls_key> APPENDING TABLE rt_contents.
```

Reply:
This is maybe an interesting case for abaplint - `SELECT * FROM (c_tabname) WHERE value = <ls_key> APPENDING TABLE rt_contents.` is a syntax error if @ is omitted on higher releases.

The universal solution is
```abap
SELECT * FROM (c_tabname)
      APPENDING TABLE rt_contents
      WHERE value = <ls_key>.
```